Components/buttons.css

```diff
@@ -0,0 +1,117 @@
+.button {
+  padding: var(--space-xs);
+  outline: none;
+  border: 0px solid transparent;
+  border-radius: 0.2rem;
+  font-family: "Merriweather Sans", sans-serif;
+  display: inline-block;
+  font-size: 1rem;
+  cursor: pointer;
+  transition: all 0.25s ease;
+}
+.button-primary {
+  color: var(--white);
+  background: var(--primary);
+}
+.button-secondary {
+  color: var(--white);
+  background: var(--secondary);
+}
+.button-success {
+  color: var(--white);
+  background: var(--success);
+}
+.button-warning {
+  color: var(--black);
```

Comment:
If except one, all buttons have white color than define white in `button` rather than in each class.

Reply:
Thanks. Didnt noticed that. Will implement